son/content/intermediate_outcomes/routes_into_work_(16_to_29_years)/entry_to_higher_education/3.0.md

```diff
@@ -0,0 +1,158 @@
+                "zkey": "secondary_split_value",
+                "scale": "%",
+                "rounding": "0dp",
+                "lowerConfidence": "lci",
+                "upperConfidence": "uci",
+                "xgrid": false,
+                "ygrid": true,
+                "xticksSmallScreen": "abreviate",
+                "yticks": 5,
+                "legend": true,
+                "colourScheme": ["#1a9e1c", "#00703c", "#eb652e", "#0096ff", "#0f265c", "#808080"],
+                "range": [0, 60],
+                "margin": [10, 20, 5, 5],
+                "maxLabelLength": 40,
+                "textLabels": "center",
+                "labelColour": "#fff"
+            }
+        :::
+    :::
+
+    ::: tab section_name="By socio-economic background" tab_name="Data"
+        ::: data_table
+            {
+                "title": "Percentage of 18 to 20 year olds enrolled in higher education, by socio-economic background (UK, 2022)",
+                "dataTable": "3.0/IN22-3.0-entry-to-higher-education--by-SEB--table-format.csv",
+                "dataTableAlignColumns": ["left", "right", "right", "right"],
+                "dataTableDecimalPlaces": [null, 1, 1, 1]
+            }
+        :::
+    :::
+
+    ::: tab section_name="By socio-economic background" tab_name="Download"
+        ::: download_section :::
+    :::
+:::
+
+---
+
+## Moving average by socio-economic background
+
+::: tabs
+    ::: tab section_name="Moving average by socio-economic background" tab_name="Visualisation"
+        #### Percentage of 18 to 20 year olds enrolled in higher education, by socio-economic background (UK, 2014 to 2024, 3-year rolling averages)
+
+        ::: visualisation vis_type="chart" section_name="By year"
+            {
+                "code": "IN35",
+                "title": "Percentage of 18 to 20 year olds enrolled in higher education, by socio-economic background (UK, 2014 to 2024, 3-year rolling averages)",
+                "tooltipTitle": "",
+                "data": "3.0/IN22-3.0-entry-to-higher-education--by-moving-average--chart-format.csv",
+                "height": 450,
+                "type": "liney",
+                "xkey": "average_window",
+                "ykey": "value",
+                "zkey": "secondary_split_value",
+                "sort": "time_period",
+                "scale": "%",
+                "rounding": "0dp",
+                "lowerConfidence": "lci",
+                "upperConfidence": "uci",
+                "xgrid": false,
+                "ygrid": true,
+                "xticksSmallScreen": "2nd",
+                "legend": true,
+                "colourScheme": ["#1a9e1c", "#00703c", "#eb652e", "#0096ff", "#0f265c", "#808080"],
+                "range": [0, 60],
+                "margin": [10, 20, 5, 5],
+                "maxLabelLength": 55,
+                "rolloverBehaviour": "fade",
+                "clickBehaviour": "isolate"
+            }
+        :::

```

Comment:
Firstly, well done for working out how to do a multi-line line chart 😄 

A couple of minor things:
* `code` should be `IN22`
(I don't think this actually does anything any more)
* This data doesn't have confidence intervals, so you can remove the `lowerConfidence` and `upperConfidence` lines.
If it did have confidence intervals, you can make them show up on the chart by adding a line `"confidenceIntervals": true,`

Reply:
done